components/molecule/textareaField/src/index.js

```diff
@@ -19,44 +18,63 @@ const getState = ({successText, errorState, alertText}) => {
   if (alertText) return AtomTextareaStates.ALERT
 }
 
-const MoleculeTextareaField = WithCharacterCount(
-  ({
-    id,
-    label,
-    maxChars,
-    textCharacters,
-    successText,
-    errorText,
-    alertText,
-    helpText,
-    onChange,
-    ...props
-  }) => {
-    const errorState = hasErrors({successText, errorText})
-    const textAreaState = getState({successText, errorState, alertText})
-
-    return (
-      <MoleculeField
-        name={id}
-        label={label}
-        textCharacters={textCharacters}
-        successText={successText}
-        errorText={errorText}
-        alertText={alertText}
-        helpText={helpText}
-        maxChars={maxChars}
-        onChange={onChange}
-      >
-        <AtomTextarea
-          id={id}
-          errorState={errorState}
-          state={textAreaState}
-          {...props}
-        />
-      </MoleculeField>
-    )
+const MoleculeTextareaField = ({
+  id,
+  label,
+  maxChars,
+  textCharacters = 'characters',
+  successText,
+  errorText,
+  alertText,
+  helpText,
+  value = '',
+  onChange = () => {},
+  ...props
+}) => {
+  const errorState = hasErrors({successText, errorText})
+  const textAreaState = getState({successText, errorState, alertText})
+
+  const [internalValue, setInternalValue] = useState(value)
+
+  const computeHelpText = () => {
```

Comment:
IMHO there is a typo `computeHelpText`. Should be `computedHelpText`

Reply:
INMHO methods are related to actions, while variables are related to names.